conf/supervisord-dev.conf

```diff
@@ -22,7 +24,13 @@ buffer_size=100
 events=PROCESS_LOG
 stderr_logfile=/dev/fd/1
 stderr_logfile_maxbytes=0
-stdout_logfile=/dev/null
+
+[program:refresh_blocklist]
+command=sh -c "while true; do bin/fetch-blocklist; sleep 60; done"
+stdout_logfile=NONE
+stderr_logfile=NONE
+stdout_events_enabled=true
+stderr_events_enabled=true
```

Comment:
:man_shrugging: Seems pretty simple. So I guess it's downloading the blocklist from AWS in dev too?

Reply:
Only if the environment variable is set, which by default it isn't.